[2025-04-01T08:40:23.372+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-04-01T06:40:00+00:00 [queued]>
[2025-04-01T08:40:23.377+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-04-01T06:40:00+00:00 [queued]>
[2025-04-01T08:40:23.378+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-01T08:40:23.386+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2025-04-01 06:40:00+00:00
[2025-04-01T08:40:23.390+0000] {standard_task_runner.py:57} INFO - Started process 148 to run task
[2025-04-01T08:40:23.392+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'transform', 'scheduled__2025-04-01T06:40:00+00:00', '--job-id', '193', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmps_urot3n']
[2025-04-01T08:40:23.393+0000] {standard_task_runner.py:85} INFO - Job 193: Subtask transform
[2025-04-01T08:40:23.403+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-01T08:40:23.424+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_pipeline.transform scheduled__2025-04-01T06:40:00+00:00 [running]> on host 07e8187327cf
[2025-04-01T08:40:23.470+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T06:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-01T06:40:00+00:00'
[2025-04-01T08:40:23.521+0000] {logging_mixin.py:154} INFO - Index(['location_id', 'aqi_us', 'main_us', 'aqi_cn', 'main_cn', 'timestamp',
       'aqi_us_category', 'aqi_cn_category', 'aqi_avg', 'main_pollutant',
       'severe_pollution', 'traffic_id', 'weather_id', 'fact_id'],
      dtype='object')
[2025-04-01T08:40:23.530+0000] {logging_mixin.py:154} INFO - ✅ Dữ liệu đã được xử lý và lưu vào processed_data
[2025-04-01T08:40:23.530+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-01T08:40:23.536+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=transform, execution_date=20250401T064000, start_date=20250401T084023, end_date=20250401T084023
[2025-04-01T08:40:23.565+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-01T08:40:23.579+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
